include/ubiquity_motor/motor_hardware.h

```diff
@@ -201,6 +204,12 @@ class MotorHardware : public hardware_interface::RobotHW {
         Joint() : position(0), velocity(0), effort(0), velocity_command(0) {}
     } joints_[2];
 
+    // MessageTypes enum for refering to motor or wheel number
+    enum MotorOrWheelNumber {
+        Motor_M1 = 1,
```

Comment:
Yes it would be the M1 and M2 are greatly ingrained in the code and it would be very messy to try to change it.   The back story here is there are numerous screw ups in right and left wheel that get sorted out at this time but are 'dirty'.   So comments in some places may refer to M1 but if we change it to L or R those comments would be disturbing.